simulatingrisk/risky_food/model.py

```diff
@@ -0,0 +1,141 @@
+from enum import Enum
+from statistics import mean
+
+import mesa
+
+
+class FoodChoice(Enum):
+    RISKY = "R"
+    SAFE = "S"
+
+
+class FoodStatus(Enum):
+    CONTAMINATED = "C"
+    NOTCONTAMINATED = "N"
+
+
+class Agent(mesa.Agent):
+    def __init__(self, unique_id, model, risk_level=None):
+        super().__init__(unique_id, model)
+        # get a random risk tolerance; returns a value between 0.0 and 1.0
+        self.risk_level = risk_level or self.random.random()
+
+    def step(self):
+        # choose food based on the probability not contaminated and risk tolerance
+        if self.risk_level > self.model.prob_notcontaminated:
+            choice = FoodChoice.RISKY
+        else:
+            choice = FoodChoice.SAFE
+        self.payoff = self.model.payoff(choice)
```

Comment:
Again I feel like a 1st-person POV might be more legible as a way of describing what's happening

Reply:
can you say more, I don't understand?

Reply:
oh, read in the wrong order - I see your comments on the other PR now